radis/levels/partfunc.py

```diff
@@ -1170,6 +1170,8 @@ def __init__(self,species):
 
     def _at(self, T):
         # Interpolate to find the partition function at the desired temperature
+        if T<10**(-5) or T>10**4:
+            raise ValueError(f"The temperature {T} is outside the accepted rannge [{10**(-5)}, {10**(4)}] K")
```

Comment:
typo `rannge`
Be more explicit, say it's because of the parition functions : 
```
raise ValueError(f"The temperature {T} is outside the accepted range [{10**(-5)}, {10**(4)}] K for the partition functions of {self.species}")
```
Also save `self.species` in __init__()